### DISCUSSION TITLE
I am curious how this sample effect is achieved

### DISCUSSION THREAD

**Discussion by Lance3d** [Mikubill/sd-webui-controlnet#1044] [Q&A] [answered]
I am curious how this sample effect is achieved, what is the model used and what are the prompts? A png file with png info would be much appreciated, thanks!

<img width="802" alt="1682480618306" src="https://user-images.githubusercontent.com/1921728/234464780-ea0c3248-0c9f-4078-ae18-3714386f2628.png">

**Comment by lllyasviel** [comment, marked as answer]
https://github.com/Mikubill/sd-webui-controlnet/issues/955

**Reply by Lance3d** [reply]
Thanks! I tried with these prompts, but the color seems to be very different... I think I used the correct model and vae.

Balanced:
![00087-808080](https://user-images.githubusercontent.com/1921728/234569787-aff70f7c-e036-4a75-971e-058ca8e28f24.png)

My Prompts is more important:
![00088-808080](https://user-images.githubusercontent.com/1921728/234569879-711c265e-62ec-4bf8-a27c-3ec231a3c3b5.png)

ControlNet is more important:
![00089-808080](https://user-images.githubusercontent.com/1921728/234569937-ba3cc18c-7dda-4b08-a022-15f1d8666414.png)

I am using ControlNet v1.1 models, could that be the problem?

**Reply by lllyasviel** [reply]
yes the parameters in this example is optimized for 1.0
